libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection/accidentDetailSubSection.ts

```diff
@@ -81,11 +82,11 @@ export const accidentDetailsSubSection = buildSubSection({
           title: accidentDetails.labels.doctorVisit,
           width: 'full',
         }),
-        buildCustomField({
+        buildDateField({
```

Comment:
Can the other custom dateField be replaced with buildDateField? If so it would be nice to get rid of that component